plugins/telemetry/prometheus.go

```diff
@@ -248,10 +252,11 @@ func (p *Plugin) updatePrometheus() {
 	}
 
 	// Update memory
-	memoryInfo, err := p.handler.GetMemory()
+	/*memoryInfo, err := p.handler.GetMemory()
```

Comment:
Perhaps some info why it was left here commented.

Reply:
That was commented by accident. Fixed now.